tests/test_pytorch/test_pytorch_ops.py

```diff
@@ -127,6 +127,27 @@ def test_instance_norm():
     assert nodes[4].domain == 'mmdeploy'
 
 
+@pytest.mark.usefixtures('prepare_symbolics_ncnn')
+class TestLinear:
+
+    def test_normal(self):
+        x = torch.rand(1, 2, 3)
+        w = torch.rand(2, 3)
+        bias = torch.rand(2)
+        model = OpModel(torch.nn.functional.linear, w, bias).eval()
+        nodes = get_model_onnx_nodes(model, x)
+        print(nodes)
+        assert nodes[0].op_type == 'Matmal'
+
+    def test_no_bias(self):
```

Comment:
```
___________________________________________ TestLinear.test_no_bias ____________________________________________

self = <test_pytorch_ops.TestLinear object at 0x7f977323a510>

    def test_no_bias(self):
        x = torch.rand(1, 2, 3)
        w = torch.rand(2, 3)
        model = OpModel(torch.nn.functional.linear, w).eval()
        nodes = get_model_onnx_nodes(model, x)
        print(nodes)
>       assert nodes[0].op_type == 'MatMul'
E       AssertionError: assert 'Constant' == 'MatMul'
E         - MatMul
E         + Constant

tests/test_pytorch/test_pytorch_ops.py:148: AssertionError
```

Reply:
Done.  And this is my output:
```
------------------------------------------------------------------------- Captured stdout call --------------------------------------------------------------------------
[input: "onnx::MatMul_0"
input: "onnx::MatMul_4"
output: "3"
name: "MatMul_0"
op_type: "MatMul"
]
```

Reply:
@tpoisonooo 
still fails on my side: pytorch==1.8.0.
the index of node is not right.
```
=============================================================== FAILURES ================================================================
________________________________________________________ TestLinear.test_normal _________________________________________________________

self = <test_pytorch_ops.TestLinear object at 0x7faebef27a50>

    def test_normal(self):
        x = torch.rand(1, 2, 3)
        w = torch.rand(2, 3)
        bias = torch.rand(2)
        model = OpModel(torch.nn.functional.linear, w, bias).eval()
        nodes = get_model_onnx_nodes(model, x)
        print(nodes)
>       assert nodes[0].op_type == 'MatMul'
E       AssertionError: assert 'Constant' == 'MatMul'
E         - MatMul
E         + Constant

tests/test_pytorch/test_pytorch_ops.py:140: AssertionError
--------------------------------------------------------- Captured stdout call ----------------------------------------------------------
[output: "1"
name: "Constant_0"
op_type: "Constant"
attribute {
  name: "value"
  t {
    dims: 3
    dims: 2
    data_type: 1
    raw_data: "\370\271\215>\236(\326>u\2321?\333\3177?\203\026\026?\260\036\262="
  }
  type: TENSOR
}
, input: "input"
input: "1"
output: "2"
name: "MatMul_1"
op_type: "MatMul"
, output: "3"
name: "Constant_2"
op_type: "Constant"
attribute {
  name: "value"
  t {
    dims: 2
    data_type: 1
    raw_data: "\344U\230> z\237="
  }
  type: TENSOR
}
, input: "2"
input: "3"
output: "4"
name: "Add_3"
op_type: "Add"
]
________________________________________________________ TestLinear.test_no_bias ________________________________________________________

self = <test_pytorch_ops.TestLinear object at 0x7faebef27d50>

    def test_no_bias(self):
        x = torch.rand(1, 2, 3)
        w = torch.rand(2, 3)
        model = OpModel(torch.nn.functional.linear, w).eval()
        nodes = get_model_onnx_nodes(model, x)
        print(nodes)
>       assert nodes[0].op_type == 'MatMul'
E       AssertionError: assert 'Constant' == 'MatMul'
E         - MatMul
E         + Constant

tests/test_pytorch/test_pytorch_ops.py:148: AssertionError
--------------------------------------------------------- Captured stdout call ----------------------------------------------------------
[output: "1"
name: "Constant_0"
op_type: "Constant"
attribute {
  name: "value"
  t {
    dims: 3
    dims: 2
    data_type: 1
    raw_data: "\267by?B{\351>\240\371\354<\207\214Y?\376\3436?%\n\033?"
  }
  type: TENSOR
}
, input: "input"
input: "1"
output: "2"
name: "MatMul_1"
op_type: "MatMul"
]


```